src/LibraryManager/Providers/jsDelivr/JsDelivrLibraryGroup.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Web.LibraryManager.Contracts;
+using Microsoft.Web.LibraryManager.LibraryNaming;
+
+namespace Microsoft.Web.LibraryManager.Providers.jsDelivr
+{
+    internal class JsDelivrLibraryGroup : ILibraryGroup
+    {
+        public JsDelivrLibraryGroup(string displayName, string description = null)
+        {
+            DisplayName = displayName;
+            Description = description;
+        }
+        public string DisplayName { get; }
+
+        public string Description { get; }
+
+        public async Task<IEnumerable<string>> GetLibraryIdsAsync(CancellationToken cancellationToken)
+        {
+
+            if (JsDelivrCatalog.IsGitHub(DisplayName).Result)
```

Comment:
Is this left empty intentionally (a TODO?) or is it meant to return empty enumerable for github?

Reply:
I think this is supposed to be empty because there's no library name autocomplete for GitHub (only versions and files). @Drgy?